test/controllers/search_controller_test.rb

```diff
@@ -210,6 +235,26 @@ def mock_timdex_search_success
     assert_select '#basic-search-main[value=data]'
   end
 
+  test 'results page shows basic USE search form when GDT is disabled' do
+    # GDT is disabled by default in test setup
+    mock_primo_search_success
+    get '/results?q=test'
+    assert_response :success
+    assert_select 'form#search-form', { count: 1 }
+    assert_select 'form#search-form-geo', { count: 0 }
+  end
+
+  test 'results page shows geo form when GDT is enabled' do
+    @test_strategy.switch!(:gdt, true)
+    mock_timdex_search_success
+    get '/results?q=test'
+    assert_response :success
+    assert_select 'form#search-form-geo', { count: 1 }
```

Comment:
Same basic question as above - should there be an assertion that `form#search-form` doesn't exist in this test?